### PR TITLE
Limit instance deletion retries

### PR DESCRIPTION
**Pull request by gsanchezu**
@EfrenRey, @GustavoLlorente, 👍?

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Deployed ElasticBox instance is available at https://10.140.45.72 and Jenkins instance at https://10.140.45.76

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/ebx_pull_request_jenkins_plugin/439/ for more details.

File: src/main/java/com/elasticbox/jenkins/model/profile/PolicyProfileType.java [hunk 1]
```diff
@@ -37,6 +43,12 @@ public ProviderType provider() {
             return ProviderType.AZURE;
         }
     },
+    AZURE_ARM("/azure-arm/arm/profile") {
+        @Override
+        public ProviderType provider() {
+            return ProviderType.AZURE;
```
Comment (EfrenRey) [inline src/main/java/com/elasticbox/jenkins/model/profile/PolicyProfileType.java]: @gsanchezu On ElasticBox ```Azure``` and ```AzureARM``` are two different provider types. Not sure if you should also define both here.

Comment (gsanchezu) [inline src/main/java/com/elasticbox/jenkins/model/profile/PolicyProfileType.java]: Ok, yes, updated adding new provider!

**Comment by allyourbotsarebelongtous** [comment]
Deployed ElasticBox instance is available at https://10.140.45.72 and Jenkins instance at https://10.140.45.76

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/ebx_pull_request_jenkins_plugin/441/ for more details.

**Comment by gsanchezu** [comment]
@GustavoLlorente, please review. I can provide you a little more context if you need. thanks!

File: src/main/java/com/elasticbox/jenkins/model/provider/ProviderType.java [hunk 2]
```diff
@@ -3,16 +3,16 @@
 public enum ProviderType {
     AMAZON,
     AZURE,
+    AZURE_ARM,
     OPENSTACK,
     SOFTLAYER,
     BYOI,
+    CENTURY_LINK,
```
Comment (EfrenRey) [inline src/main/java/com/elasticbox/jenkins/model/provider/ProviderType.java]: Minor comment. I would use ```CENTURYLINK```, but as you want.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See https://jenkins.elasticbox.com/job/ebx_pull_request_jenkins_plugin/442/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See https://jenkins.elasticbox.com/job/ebx_pull_request_jenkins_plugin/443/ for more details.

File: src/main/java/com/elasticbox/jenkins/model/provider/ProviderType.java (hunk 2 above)
Comment (gsanchezu) [inline src/main/java/com/elasticbox/jenkins/model/provider/ProviderType.java]: 👍

**Comment by gsanchezu** [comment]
Jenkins test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See https://jenkins.elasticbox.com/job/ebx_pull_request_jenkins_plugin/444/ for more details.

**Comment by gsanchezu** [comment]
Jenkins test this please 2

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See https://jenkins.elasticbox.com/job/ebx_pull_request_jenkins_plugin/445/ for more details.

**Comment by gsanchezu** [comment]
Jenkins test this please 3

**Comment by allyourbotsarebelongtous** [comment]
Deployed ElasticBox instance is available at https://10.166.246.58 and Jenkins instance at https://10.166.246.49

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See https://jenkins.elasticbox.com/job/ebx_pull_request_jenkins_plugin/446/ for more details.

**Comment by GustavoLlorente** [comment]
ACK

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-4xeob8, https://ops.elasticbox.com/services/instances/i-vdye4g

**Pull request merged**